iOS/JSQMessagesViewController/component/component.yaml

```diff
@@ -18,6 +18,10 @@ libraries:
 samples: 
   - name: "Xamarin Chat Sample"
     path: ../samples/XamarinChat.sln
+    installNuGets:
+      - project: XamarinChat
+        packages:
+          - Xamarin.JSQMessagesViewController
     configuration: "Release|iPhone"
 no_build: true

```

Comment:
No externals file being included in the xam package

Reply:
Do you mean the Third Party Notice?